### DISCUSSION TITLE
New Ring security instal not syncing with hoobs or homekit.

### DISCUSSION THREAD

**Discussion by dali341** [dgreif/ring#828] [General]
Hey everyone, newbie here. I installed ring app a few weeks ago and installed a ring doorbell and everything works and synced with home kit. Today i installed a ring security set up but nothing is syncing with hoobs or home kit. Am i missing something? Do i need to get a new refresh token every time?

**Comment by dgreif** [comment]
You should just need to restart your homebridge server
